src/app/content/highlights/hooks/createHighlight.ts

```diff
@@ -1,9 +1,9 @@
 import { ActionHookBody } from '../../../types';
-import { actionHook } from '../../../utils';
+import { actionHook, makeApiCallOrThrow } from '../../../utils';
 import { createHighlight } from '../actions';
 
 export const hookBody: ActionHookBody<typeof createHighlight> = ({highlightClient}) => async({payload}) => {
-  await highlightClient.addHighlight({highlight: payload});
+  await makeApiCallOrThrow(highlightClient.addHighlight({highlight: payload}));
```

Comment:
is it possible to add this to the fetch call in `src/gateways/createHighlightClient.ts` instead of in every place the api is called from?